openshift/multi-project-templates/user-rolebinding.yml

```diff
@@ -27,7 +27,7 @@ objects:
 - apiVersion: rbac.authorization.k8s.io/v1
   kind: RoleBinding
   metadata:
-    name: ${CLUSTER_ROLE}-${USER}
+    name: ${CLUSTER_ROLE}-${USER}-${NAMESPACE}
```

Comment:
Role bindings exist inside of namespaces, so the namespace name is not necessary.